download/releaseNotes/releaseNotes7.0.adoc

```diff
@@ -195,3 +195,82 @@ This example also demonstrates real-time producing and consuming of tasks.
 * The documentation is now in AsciiDoc, which improves output quality and makes it easier to contribute.
 * `HardMediumSoftBigDecimalScore`: 3 score levels of BigDecimal scores. Contributed by Brad Hards.
 * The public API is now a solid foundation for the 7.x years.
+
+== Workbench
+
+=== Enhanced solver editor
+
+OptaPlanner Solver editor screen now supports adding all termination types, including composite termination.
+Phase configuration section allows you to tweak Construction Heuristic settings and select Local Search alogorithm
```

Comment:
typo alogorithm
+ change "allow you" to avoid the you form by stating "allows to tweak" or "can now tweak"